src/Python/qsharp-core/qsharp/tests/test_iqsharp.py

```diff
@@ -85,6 +163,28 @@ def test_multi_compile():
     assert r == qsharp.Result.One
 
 
+def test_config():
+    """
+    Verifies get and set of config settings
+    """
+    config = qsharp.get_config()
+    assert len(config) == 0
```

Comment:
Will those go badly if the test environment somehow has a `.iqsharp-config.json` file?

Reply:
Yes, good point. With the refactoring into a separate `Config` object, I don't think these checks really add much value, so I'll just remove them.